src/components/Header/header.css

```diff
@@ -0,0 +1,81 @@
+.btn {
+  border-style: none;
+  color: #fff;
+  background-color: #2d6180;
+  padding: 0 20px;
+}
+
+.header {
+  width: 100%;
+  height: 3rem;
+  padding: 0 1.4rem;
```

Comment:
padding nên dùng px thay vì rem. rem chỉ nên dùng cho font-size
[ref](https://medium.com/@sascha.wolff/dont-use-rem-em-for-paddings-margins-and-more-94e19026b000)